integration_tests/tests.ts

```diff
@@ -1,19 +1,25 @@
 import { payload, snsParams, publishSNS } from "./helpers/snsHelper";
-import { getFilteredEventFromLatestLogStream, getLogGroupName } from "./helpers/cloudWatchHelper";
+import {
+  getFilteredEventFromLatestLogStream,
+  getLogGroupName,
+} from "./helpers/cloudWatchHelper";
 import { PublishResponse } from "@aws-sdk/client-sns";
 import { scanDB } from "./helpers/dynamoDBHelper";
+import delay from "delay";
 
 let snsResponse: PublishResponse;
 
-//below E2E tests is marked as skipped as still development in progress
-describe.skip("E2E tests", () => {
+describe("E2E tests", () => {
   beforeAll(async () => {
     snsResponse = await publishSNS(snsParams);
     expect(snsResponse).toHaveProperty("MessageId");
   });
   test("Publish sns message and expect message to reach dynamoDB ", async () => {
+    await delay(5000);
```

Comment:
It probably doesn't make a lot of difference with the small number of tests we have but a polling system with a timeout might be better than delaying for a number of seconds - i.e. each time we want to check if something exists in the DB we check (say) every five seconds until it is true and time out after (say) thirty seconds.

Reply:
fixed, added while condition to check for the event in the DB with timeout